lib/Network/OAuth2/Server/Configuration.hs

```diff
@@ -39,15 +42,16 @@ import           Network.Wai.Middleware.Shibboleth as S
 -- You'll want to set optDBString at minimum.
 defaultServerOptions :: ServerOptions
 defaultServerOptions =
-    let optDBString = ""
-        optStatsHost = "localhost"
-        optStatsPort = 8888
-        optServiceHost = "*"
-        optServicePort = 8080
-        optUIPageSize = (10 :: Integer) ^?! pageSize
-        optVerifyRealm = "verify-token"
-        optShibboleth = S.defaultConfig
-    in ServerOptions{..}
+    let optDBString     = ""
+        optStatsHost    = "localhost"
+        optStatsPort    = 8888
+        optServiceHost  = "*"
+        optServicePort  = 8080
+        optUIPageSize   = (10 :: Integer) ^?! pageSize
+        optUIStaticPath = unsafePerformIO P.getDataDir </> "static"
```

Comment:
This is supposed to be a complete, coherent, "working" default value. I'd probably prefer to remove it completely than make it a function or put it in `IO`.